examples/camera-app/linux/src/clusters/webrtc-requestor/webrtc-requestor-manager.cpp

```diff
@@ -0,0 +1,526 @@
+/*
+ *
+ *    Copyright (c) 2025 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "webrtc-requestor-manager.h"
+
+#include <Options.h>
+#include <app/server/Server.h>
+#include <camera-av-stream-manager.h>
+#include <controller/InvokeInteraction.h>
+#include <iostream>
+#include <lib/support/CHIPFaultInjection.h>
+#include <lib/support/logging/CHIPLogging.h>
+#include <webrtc-transport.h>
+
+using namespace chip;
+using namespace chip::app;
+using namespace chip::app::Clusters;
+using namespace chip::app::Clusters::WebRTCTransportRequestor;
+
+namespace {
+
+// Constants
+constexpr int kVideoH264PayloadType = 96; // 96 is just the first value in the dynamic RTP payload‑type range (96‑127).
+constexpr int kVideoBitRate         = 3000;
+
+const char * GetPeerConnectionStateStr(rtc::PeerConnection::State state)
+{
+    switch (state)
+    {
+    case rtc::PeerConnection::State::New:
+        return "New";
+
+    case rtc::PeerConnection::State::Connecting:
+        return "Connecting";
+
+    case rtc::PeerConnection::State::Connected:
+        return "Connected";
+
+    case rtc::PeerConnection::State::Disconnected:
+        return "Disconnected";
+
+    case rtc::PeerConnection::State::Failed:
+        return "Failed";
+
+    case rtc::PeerConnection::State::Closed:
+        return "Closed";
+    }
+    return "N/A";
+}
+
+const char * GetGatheringStateStr(rtc::PeerConnection::GatheringState state)
+{
+    switch (state)
+    {
+    case rtc::PeerConnection::GatheringState::New:
+        return "New";
+
+    case rtc::PeerConnection::GatheringState::InProgress:
+        return "InProgress";
+
+    case rtc::PeerConnection::GatheringState::Complete:
+        return "Complete";
+    }
+    return "N/A";
+}
+
+} // namespace
+
+void WebRTCRequestorManager::Init(ScopedNodeId peerId, EndpointId endpointId)
+{
+    ChipLogProgress(Camera, "WebRTCRequestorManager::Init");
+
+    // PeerId and Endpoint of the controller
+    mPeerId                = peerId;
+    mOriginatingEndpointId = endpointId;
+
+    rtc::InitLogger(rtc::LogLevel::Error);
+
+    rtc::Configuration config;
+    // config.iceServers.emplace_back("stun.l.google.com:19302");
```

Comment:
Why is this commented out?  Looks like hard coding of a test server, is there a need to be able to do this programmatically in the code?

Reply:
Thank you for your review. This is not needed currently, removing this